saleor/graphql/shipping/mutations/shippings.py

```diff
@@ -154,7 +154,7 @@ def clean_default(cls, instance, data):
                 )
             else:
                 countries = get_countries_without_shipping_zone()
-                data["countries"] = countries
+                data["countries"].extend([country for country in countries])
```

Comment:
Is `data["countries"]` always defined and is it always a list? Just wondering if `extend` wouldn't fail in some cases like when countries are not passed to the mutation.

Reply:
Yes, it is always a list (defined as graphene.List input type field). If no country is passed it is an empty list.